From 463da729537faa6d5ef206912fdec871c1059dac Mon Sep 17 00:00:00 2001
From: Petr Vorel <pvorel@suse.cz>
Date: Thu, 16 Jun 2022 16:33:31 +0200
Subject: [PATCH] nfs05_make_tree: Restore 5 min timeout

nfs05_make_tree is a long running test. The previous default timeout 5
min was enough after reducing runtime in 05be61cf6.

But the new default 30 sec introduced in the concept of max runtime is
definitely not enough. Restore the previous timeout 5 min.

Signed-off-by: Petr Vorel <pvorel@suse.cz>

[ upstream status: 463da729537faa6d5ef206912fdec871c1059dac ]
---
 testcases/network/nfs/nfs_stress/nfs05_make_tree.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/testcases/network/nfs/nfs_stress/nfs05_make_tree.c b/testcases/network/nfs/nfs_stress/nfs05_make_tree.c
index fdc13bd5d..5456c1bf0 100644
--- a/testcases/network/nfs/nfs_stress/nfs05_make_tree.c
+++ b/testcases/network/nfs/nfs_stress/nfs05_make_tree.c
@@ -215,4 +215,5 @@ static struct tst_test test = {
 	.options = opts,
 	.test_all = do_test,
 	.setup = setup,
+	.max_runtime = 300,
 };
-- 
2.37.1

